### DISCUSSION TITLE
faster-whisper implementation

### DISCUSSION THREAD

**Discussion by jooni22** [absadiki/subsai#13] [Ideas]
Hi, you should think about faster-whisper implementation. IMHO the best way in speech to transcribe method right now.
https://github.com/guillaumekln/faster-whisper

**Comment by absadiki** [comment]
Hi @jooni22,

Thanks for bringing it to my attention. `faster-whisper` seems really good. 
I will add it as soon as possible.

Thanks!

**Comment by absadiki** [comment]
Hi @jooni22,

Support for `faster-whisper` has been added. 
Let me know if you find any issues. 

Thanks again!

**Reply by shubhendumadhukar** [reply]
Anyway I can enable logging for faster-whisper? While using whisper or whispercpp, I can see the progress/percentage. But not the case with faster-whisper? No error, so I am not sure if it's stuck. I am new to this, not really sure if a progress bar is supported in faster-whisper implementation.

**Reply by absadiki** [reply]
@shubhendumadhukar, are you using faster-whisper from Python or the web-ui ?

**Reply by shubhendumadhukar** [reply]
I'm using it from the web ui.

**Reply by absadiki** [reply]
@shubhendumadhukar I don't think faster-whisper provides a helpful progress as other models. But I have activated debugging level. It shows the time of the segment being processed. Update `subsai` to the newest commit and give it a try.

**Comment by jooni22** [comment]
Very fast implementation I will test the new model then let you know.
